entity-management-tests/src/integration-test/java/eu/europeana/entitymanagement/testutils/IntegrationTestUtils.java

```diff
@@ -247,20 +248,24 @@ public class IntegrationTestUtils {
           ORGANIZATION_PCCE_ZOHO_RESPONSE,
           ORGANIZATION_BERGER_MUSEUM_URI_ZOHO,
           ORGANIZATION_BERGER_MUSEUM_ZOHO_RESPONSE);
-
+  
   /** Maps Metis dereferenciation URIs to mocked XML responses */
-  public static final Map<String, String> METIS_RESPONSE_MAP =
-      Map.of(
-          AGENT_DA_VINCI_URI, AGENT_DA_VINCI_XML,
-          AGENT_STALIN_URI, AGENT_STALIN_XML,
-          PLACE_PARIS_URI, PLACE_PARIS_XML,
-          PLACE_HAGENBACH_URI, PLACE_HAGENBACH_XML,
-          TIMESPAN_1ST_CENTURY_URI, TIMESPAN_1ST_CENTURY_XML,
-          CONCEPT_BATHTUB_URI, CONCEPT_BATHTUB_XML,
-          AGENT_JAN_VERMEER_VIAF_URI, AGENT_JAN_VERMEER_XML_VIAF,
-          AGENT_JAN_VERMEER_WIKIDATA_URI, AGENT_JAN_VERMEER_XML_WIKIDATA,
-          AGENT_SCHEGK_URI, AGENT_SCHEGK_XML,
-          AGENT_BIRCH_URI, AGENT_BIRCH_XML);
+  public static final Map<String, String> METIS_RESPONSE_MAP = new HashMap<>(){
```

Comment:
Because the Map.of is limited to only 10 entries, please see here: https://typeofnan.dev/how-to-create-a-map-of-more-than-10-in-java/. It can also be written the way I did it, but I changed it as it is in the article, with Map.ofEntries.